Code/Legacy/CryCommon/platform.h

```diff
@@ -7,136 +7,31 @@
  */
 
 
-// Description : Platform dependend stuff.
+// Description : Platform dependent stuff.
 //               Include this file instead of windows h
 
 #pragma once
 
 #if defined(AZ_RESTRICTED_PLATFORM)
 #undef AZ_RESTRICTED_SECTION
-#define PLATFORM_H_SECTION_1 1
```

Comment:
Ultimately this whole file should go away, but I would leave all these PLATFORM_H_SECTION_* s in place in case they're being used by one or more restricted platforms (which is likely).

Reply:
the rest of the file is being used in one way or another, mostly within CryCommon and CrySystem. I removed the PLATFORM_H_SECTION_1 that no longer exist, but did not touch the different restricted pieces
This likely will be the last file to go away for the cry removal since it is what most cry files include

Reply:
Yeah, I know the whole file won't be going away anytime soon, was just prefacing the comment to indicate that it doesn't overly matter what we do with it in the meantime. My concern was that one or more of the PLATFORM_H_SECTION_* sections you've removed may be required by one or more restricted platforms, but I checked and that doesn't appear to be the case, with the exception of PLATFORM_H_SECTION_15, so maybe just leave that one in for now.

Reply:
Putting that back